core/src/main/java/com/linecorp/armeria/common/stream/FilteredStreamMessage.java

```diff
@@ -66,7 +66,7 @@ protected FilteredStreamMessage(StreamMessage<T> delegate) {
     protected FilteredStreamMessage(StreamMessage<T> delegate, boolean withPooledObjects) {
         requireNonNull(delegate, "delegate");
         this.delegate = delegate;
```

Comment:
nit: Not a part of your PR, but please change here to `this.delegate = requireNonNull(delegate, "delegate");` for simplicity. 🙇